src/Microsoft.ML.Data/Training/EarlyStoppingCriteria.cs

```diff
@@ -62,7 +62,7 @@ public abstract class ArgumentsBase { }
             }
         }
 
-        internal EarlyStoppingCriterion(TArguments args, bool lowerIsBetter)
+        internal EarlyStoppingCriterion(TOptions args, bool lowerIsBetter)
```

Comment:
>args [](start = 49, length = 4)

Could you rename the local variable names as well?

Reply:
fixed.

fyi.. in this PR the focus is scrubbing the public surface area for  `Options options` .   

There are still  lots of  `internal`  occurrences of `Arguments args` .. Those  are  P2  since they can be renamed post V1.0 as well

---
In reply to: [257144702](https://github.com/dotnet/machinelearning/pull/2563#discussion_r257144702) [](ancestors = 257144702)